x/checkers/types/full_game.go

```diff
@@ -0,0 +1,54 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/xavierlepretre/checkers/x/checkers/rules"
+)
+
+type FullGame struct {
```

Comment:
what is the reason for having two types?

Reply:
For instance, later on, like [here](https://github.com/cosmos/b9-checkers-academy-draft/pull/12/files#diff-5fec09d1b6ea66c9a28af8aad82729eb603eca996ead9d5d5691422473ba23c4R98), it allows us to use complex types, which make life easier.
Do you think I can achieve the same with extension methods of `StoredGame`?